packages/paste-core/components/time-picker/__tests__/index.spec.tsx

```diff
@@ -24,11 +24,11 @@ describe('formatReturnTime()', () => {
     expect(formatReturnTime(TEST_TIME, 'hh:mm:ss')).toEqual('01:30:04');
     expect(formatReturnTime(TEST_TIME, 'HH:mm:ss.SSS')).toEqual('13:30:04.000');
     expect(formatReturnTime(TEST_TIME, 'HH:mm:ss.SSS')).toEqual('13:30:04.000');
-    expect(formatReturnTime(TEST_TIME, 'hh:mm:ss.SSS aa')).toEqual('01:30:04.000 p.m.');
+    expect(formatReturnTime(TEST_TIME, 'hh:mm:ss.SSS aa')).toEqual('01:30:04.000 PM');
```

Comment:
Question: were these tests failing that you fixed?

Reply:
Yep. Those started failing after I added the missing dep.